sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -419,6 +428,32 @@ export class KeyClient {
     });
   }
 
+  /**
+   * Gets a {@link CryptographyClient} for the given key.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(url, credentials);
+   * // get a cryptography client for a given key
+   * let cryptographyClient = client.getCryptographyClient("MyKey");
+   * ```
+   * @param name - The name of the key used to perform cryptographic operations.
+   * @param version - Optional version of the key used to perform cryptographic operations.
+   * @returns - A {@link CryptographyClient} using the same options, credentials, and http client as this {@link KeyClient}
+   */
+  public getCryptographyClient(keyName: string, keyVersion?: string): CryptographyClient {
```

Comment:
Before merging I wanted to check with @bterlson or @chradek whether they prefer

```ts
getCryptographyClient(keyName: string, keyVersion?: string)
```

or 

```ts
getCryptographyClient(keyName: string , { keyVersion?: string })
```
I believe most of our keyClient methods take the latter but in this case I wasn't sure what makes more sense - there's no other options there other than this one lonely version...

Reply:
I prefer the second if there is ever a chance we'll add more optional options later, but maybe that's playing it too safe?

Reply:
Shoot I didn't notice this. I can change it. It's honestly not that huge a change from a caller's perspective so we might as well stick to the guidelines

Reply:
Updated to use options and re-requested your review when you get a chance